2021q4/ena.adoc

```diff
@@ -0,0 +1,26 @@
+=== ENA FreeBSD Driver Update
+
+Links: +
+link:https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README[ENA README] URL: link:https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README[https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README]
+
+Contact: Michal Krawczyk <mk@semihalf.com> +
+Contact: Dawid Gorecki <dgr@semihalf.com> +
+Contact: Marcin Wojtas <mw@semihalf.com>
+
+ENA (Elastic Network Adapter) is the smart NIC available in the virtualized environment of Amazon Web Services (AWS).
+The ENA driver supports multiple transmit and receive queues and can handle up to 100 Gb/s of network traffic, depending on the instance type on which it is used.
+
+Completed since the last update:
+
+* Add IPv6 L4 checksum offload support to the driver
+* Add NUMA awareness to the driver whenever kernel RSS option is being enabled
```

Comment:
"when the RSS kernel option is enabled", I think.

Reply:
Done